packages/realm/src/Realm.ts

```diff
@@ -285,6 +285,7 @@ export class Realm {
 
   /**
    * Clears the state by closing and deleting any Realm in the default directory and logout all users.
+   * NOTE: Not a part of the public API and it's primarily used from the library's tests.
    * @private
    */
   public static clearTestState(): void {
```

Comment:
Regarding `clearTestState()`, do you know of the reason why this is public with the private annotation, rather than internal?

Reply:
Because it needs to be callable from the integration tests. All the stuff we annotate internal is completely stripped from the types from the POV of the integration tests.

Reply:
Thought we usually used `// @ts-expect-error` when having to use our internal parts (since it's still callable despite `@internal`). But helps with intellisense to not have it.